editor.planx.uk/src/pages/FlowEditor/components/Sidebar/index.tsx

```diff
@@ -72,15 +72,27 @@ const Header = styled("header")(({ theme }) => ({
     flex: "1",
     padding: "5px",
     marginRight: "5px",
-    background: theme.palette.common.white,
+    background: theme.palette.background.paper,
+
     border: "1px solid rgba(0, 0, 0, 0.2)",
   },
-  "& svg": {
-    cursor: "pointer",
-    opacity: "0.7",
-    margin: "6px 4px 1px 4px",
-    fontSize: "1.2rem",
-  },
+}));
+
+const SecondaryButton = styled(IconButton)(({ theme }) => ({
+  color: theme.palette.common.black,
+}));
+
+const ViewButton = styled(Button)(({ theme }) => ({
+  background: theme.palette.common.white,
+  border: `1px solid ${theme.palette.border.main}`,
+  boxShadow: "none",
+  color: theme.palette.common.black,
+  width: "33%",
+  display: "flex",
+  flexDirection: "row",
+  gap: "8px",
+  borderRadius: "5px",
+  padding: "8px",
```

Comment:
nit: Feels a bit surprising & brittle to me to see properties like `display`, `flexDirection`, and `gap` defined directly on a `Button` - I think I'd expect to see a wrapper `Box`/`div` with these properties that then accepts buttons as children ??